ethereum_history_api/circuits/lib/src/misc/fragment.nr

```diff
@@ -60,10 +60,9 @@ impl<MAX_DATA_LEN, T> Fragment<MAX_DATA_LEN, T> {
     }
 
     pub fn at(self, index: u64) -> T {
-        let length = self.length;
-        assert(
-            index < self.length, f"Index out of bounds, tried to access index {index} but the length is {length}"
-        );
+        // Previously, the assert message was f"Index out of bounds, tried to access index {index} but the length is {length}".
+        // As of nargo 0.29.0 replacing the f-string with normal string prevents compiler panic e.g. on header_int_test. 
```

Comment:
Please add info about the compiler panic message and that we've made the attempt to create a repro before giving up